src/Common/Utils.fs

```diff
@@ -0,0 +1,16 @@
+module Common.Utils
+
+open Microsoft.FSharp.Collections
+open Microsoft.FSharp.Reflection
+
+let fromString<'a> (s:string) =
```

Comment:
By the way, why do we return an option here? I would suppose empty array can't be returned from `GetUnionCases` because DU must have at least one member...